lessons/prework/codewars-signup.md

```diff
@@ -0,0 +1,23 @@
+**In order to signup for a CodeWars account you'll need to do the following:**
+
+- Go to [codewars.com](https://www.codewars.com/) and click the 'Sign Up' button in the upper right corner
+- It will then ask you to 'Choose your language to begin...' Select 'JavaScript' to continue.
+- You should then be prompted to solve two chllanges. A couple hints: the first problem is missing a `return` statement. The second problem is missing a `this` keyword, which we have not covered yet, so the soluttion is provided for you below:
+
+```javascript 
+function Person(name){
+  this.name = name;
+}
+
+Person.prototype.greet = function(otherName){
+  return "Hi " + otherName + ", my name is " + this.name;
+}
+```
+
+- You should then be prompted to sign up by entering your email and username. Do not worry about entering a 'Clan' (aka company/school/organization) - you can leave that blank for now. Once you've entered your email, username, and password, hit 'Enlist.'
```

Comment:
Was debating it...Rina thinks we should hold off on Clan until we start class so we don't intimidate people when they're true beginners